app/engine/RowingStatistics.js

```diff
@@ -144,13 +147,17 @@ function createRowingStatistics (config, session) {
     // We need to emit the metrics BEFORE the sessionstatus changes to anything other than "Rowing", as it forces most merics to zero
     emitMetrics('rowingStopped')
     sessionStatus = 'Stopped'
+    log.debug('stopTraning')
```

Comment:
apologies. I left a lot of debugging logging code in. I should have cleaned this up before submitting